src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs

```diff
@@ -602,11 +602,16 @@ private void StartNewTasMovie()
 				Config.DefaultAuthor);
 
 			SetTasMovieCallbacks(tasMovie);
+			((MovieSession) MovieSession).SetMovieController(Emulator.ControllerDefinition); //HACK

```

Comment:
Why not expose it in the interface

Reply:
Since this is a hack that ends up coming from TAStudio's weird use of default.tasproj I would assume it's better to not expose it in the interface and only use it directly (I would assume getting rid of the default.tasproj usage would have this remove?)

Reply:
Decided to expose it in the interface anyways.